src/pages/MainPage/HeroSection.jsx

```diff
@@ -1,18 +1,21 @@
-import SearchPannel from '../../components/common/SearchPannel/SearchPannel';
-import './heroSection.scss';
+import SearchPanel from '../../components/common/SearchPanel/SearchPanel';
```

Comment:
You pushed the fix and changed file name, but the folder name is still the same containing a typo

Have you been running project locally? Because I immediately get an error.

Reply:
<img width="1509" alt="image" src="https://github.com/voyageglobal/voyage-frontend/assets/19875209/08beeb2b-572d-40a1-a87f-2bdb0069b7ee">